rachellegelden/Home-Together-Canada
# Htc 87/member profile static form

# [87](https://github.com/rachellegelden/Home-Together-Canada/issues/87)

## Summary
Adding static form for member profile information. Additional functional components added, and snapshot tests + snapshots for each.

## Relevant Motivation & Context
Needed for feature.

## Testing Instructions
npm test in root - all should pass
npm start in /client - see what breaks - the logic is incomplete - radio buttons especially are giving me a hard time
Inspect code - make suggestions. The more precise the better, realizing my time is not infinite.

## Developer checklist prior to opening this pull request:


## Reviewer
